pyrs/interface/manual_reduction/event_handler.py

```diff
@@ -51,67 +51,6 @@ def _set_sub_run_numbers(self, sub_runs):
         for sub_run in sorted(sub_runs):
             self.ui.comboBox_sub_runs.addItem('{}'.format(sub_run))
 
-    def browse_calibration_file(self):
```

Comment:
Some of these methods, if not all, are used.

Reply:
Thanks, let's talk about adding tests tomorrow. Currently these lines of code are.not covered in CI.